sdcm/cluster.py

```diff
@@ -1958,6 +1958,15 @@ def install_scylla_debuginfo(self):
             self.remoter.run(r'sudo apt-get install -y -o Dpkg::Options::="--force-confdef" -o Dpkg::Options::="--force-confold" --force-yes --allow-unauthenticated {0}-server-dbg={1}\*'
                              .format(self.scylla_pkg(), self.scylla_version), ignore_status=True)
 
+    def is_scylla_install(self):
+        if self.distro.is_rhel_like:
+            result = self.remoter.run('yum list installed scylla', verbose=False, ignore_status=True)
```

Comment:
Replace `scylla` with return value of `self.scylla_pkg()`. In other case, it'll not work with `scylla-enterprise` package.

Also, it's better to use `rpm -q ...` command instead of `yum`.

Reply:
fixed